### ISSUE TITLE
expand_action can't handle service *

### ISSUE THREAD

**Issue by lingrlongr** [duo-labs/parliament#28]
When given an admin policy (`*:*`), `expand_action` is not able to produce a full list of actions.  Error produced is:
```
    parliament.UnknownPrefixException: Unknown prefix *
```

**Comment by 0xdabbad00** [comment]
I have a special case before `expand_action` is called in the code to account for those, but I shouldn't force users of this library to do that so I'll make a fix for this. https://github.com/duo-labs/parliament/blob/53d8a7640911047dd82f49cd77d719d9457c96ad/parliament/statement.py#L291

**Comment by lingrlongr** [comment]
Yea, I was toying with the idea of updating [this](https://github.com/duo-labs/parliament/blob/master/parliament/__init__.py#L219) line with:
```
    or prefix == '*'
```

**Comment by 0xdabbad00** [comment]
Fixed in #38
